generators/pythonv2/codegen/src/ast/CodeBlock.ts

```
@@ -0,0 +1,22 @@
+
+export class CodeBlock extends AstNode {
+    private arg: CodeBlock.Arg;
+
+    public constructor(arg: CodeBlock.Arg) {
+        super();
+        this.arg = arg;
+    }
+
+    public write(writer: Writer): void {
+        const commonCodeBlock = new CommonCodeBlock(this.arg);
+        return commonCodeBlock.write(writer);
```

Biome

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)